lib/iris/tests/idiff.py

```diff
@@ -86,7 +86,8 @@ def diff_viewer(repo, key, repo_fname,
         sha1 = hashlib.sha1(fi.read())
     result_dir = os.path.dirname(result_fname)
     fname = sha1.hexdigest() + '.png'
-    uri = os.path.join(_PREFIX_URI, fname)
+    base_uri = 'https://scitools.github.io/test-images-scitools/image_files'
+    uri = os.path.join(base_uri, fname)
```

Comment:
I agree. even M$ windows respects the HTTP protocol's use of `/` as a separator in all circumstances.  This should always return a `/` separator, so the use of `os.path.join` is incorrect